ui/src/dev/server/Dockerfile

```diff
@@ -2,3 +2,5 @@ FROM node:alpine
 
 # Java for Swagger client generation
 RUN apk --no-cache add openjdk8-jre
+
+RUN mkdir /.config &&  mkdir /.npm && chmod a+rwx /.config /.npm
```

Comment:
I'm getting an `EACCESS` error here:

```
+ docker-compose run --rm ui npm install
WARNING: The ENV_FLAG variable is not set. Defaulting to a blank string.
npm info it worked if it ends with ok
npm info using npm@5.3.0
npm info using node@v8.5.0
npm info lifecycle aou-ui-angular@0.0.2~preinstall: aou-ui-angular@0.0.2
npm ERR! path /.npm
npm ERR! code EACCES
npm ERR! errno -13
npm ERR! syscall mkdir
npm ERR! Error: EACCES: permission denied, mkdir '/.npm'
npm ERR!  { Error: EACCES: permission denied, mkdir '/.npm'
npm ERR!   stack: 'Error: EACCES: permission denied, mkdir \'/.npm\'',
npm ERR!   errno: -13,
npm ERR!   code: 'EACCES',
npm ERR!   syscall: 'mkdir',
npm ERR!   path: '/.npm' }
npm ERR!
npm ERR! Please try running this command again as root/Administrator.
```

Honestly I had expected this error (or something like it) on the API side as well but its not occurring there.  What does seems odd about the server side is the presence of two new directories: `api/?/` and `/api/db/?/`.